src/testing_system/internal/controller/Orchestrator.py

```
@@ -0,0 +1,87 @@
+            logger.info(f"Orchestrator: Experiment '{experiment.id}:{experiment.name}' is complete in {experiment.finished_at}")
+
+
+    def _choose_assistant(self, cfg: dict) -> IAssistant:
+        if cfg["type"] == "ollama":
+            return OllamaAssistant(
+                model=cfg["ollama"]["model"], 
+                base_url=f"http://{cfg['ollama']['host']}:{cfg['ollama']['port']}"
+                )
+        elif cfg["type"] == "cloud":
+            raise NotImplementedError
+            return OpenAIAssistant(
```

Ruff (0.15.12)

[error] Undefined name `OpenAIAssistant`

(F821)